### PR TITLE
Move survey banner to header

### PR DESCRIPTION
**Pull request by actlikewill**
image

image

image

image

### CODE REVIEW

**Comment by actlikewill** [comment]
@longhotsummer its now smaller on the other pages

**Comment by longhotsummer** [comment]
I think it's time to make the survey a template that can be `included` and overridden.

You could also make this more configurable with a peachjam settings for `survey_link` and show the banners if that's configured.

File: malawilii/templates/liiweb/home.html [hunk 1]
```diff
@@ -17,23 +18,21 @@ <h4 class="mb-4">
...
-            <!-- text -->
-            <p class="text-white mb-4 h5 fw-bold">
-              Help us make Malawi legal information more freely accessible to everyone.
-              Take this short 5 minute survey on your usage of the platform and your general experience using the site.
-              All information shared is confidential.
-            </p>
+            <h1 class="text-white fw-bold pe-lg-8">{% trans 'Take the MalawiLII Survey' %}</h1>
+            {% blocktrans %}
+              <p class="text-white mb-4 h5 fw-bold">
+                Help us make Malawi legal information more freely accessible to everyone.
```
Comment (longhotsummer) [inline malawilii/templates/liiweb/home.html:29]: You can template this more easily for re-use, and tighten the wording:

```
Help us make APP_NAME better. Take this short 5 minute survey and tell us about how you use APP_NAME and how we can improve it. All information shared is confidential.
```

Comment (longhotsummer) [reply malawilii/templates/liiweb/home.html:29]: I think you can safely re-use the same wording in both banners.

File: peachjam/templates/peachjam/_survey_banner_sm.html [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{% load i18n %}
+<div class="bg-primary py-3">
+  <div class="container">
+    <div class="row align-items-center g-0">
+      <div class="col-xl-6 col-lg-6 col-md-12">
+        <div>
+          {% blocktrans %}
+            <p class="text-white m-lg-0 fw-bold">
+             Help us make {{ APP_NAME }} better. Take this short 5 minute survey and tell us about how you use {{ APP_NAME }} and how we can improve it. All information shared is confidential
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_survey_banner_sm.html:9]:
```suggestion
             Help us make {{ APP_NAME }} better. Take this short 5 minute survey and tell us about how you use {{ APP_NAME }} and how we can improve it. All information shared is confidential.
```

File: peachjam/templates/peachjam/_survey_banner_lg.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+               Take this short 5 minute survey and tell us about how you use {{ APP_NAME }} and how we can improve it.
+               All information shared is confidential.
+              </p>
+            {% endblocktrans %}
+        </div>
+      </div>
+      <div class=" col-xl-6 col-lg-6 col-md-12 text-lg-center pt-6">
+        <a target="_blank"
+           href="{{ PEACHJAM_SETTINGS.survey_link }}"
+           class="text-white btn btn-lg btn-danger"><span class="fw-bold">Take the survey</span></a>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_survey_banner_lg.html:20]: translateable

File: peachjam/templates/peachjam/_survey_banner_sm.html [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            </p>
+          {% endblocktrans %}
+        </div>
+      </div>
+      <div class=" col-xl-6 col-lg-6 col-md-12 text-lg-center pt-6">
+        <a target="_blank"
+           href="{{ PEACHJAM_SETTINGS.survey_link }}"
+           class="text-white btn btn-danger"><span class="fw-bold">Take the survey</span></a>
```
Comment (longhotsummer) [inline peachjam/templates/peachjam/_survey_banner_sm.html:17]: translateable

File: peachjam/templates/peachjam/_survey_banner_sm.html (hunk 2 above)
Comment (actlikewill) [reply peachjam/templates/peachjam/_survey_banner_sm.html]: changed

File: peachjam/templates/peachjam/_survey_banner_lg.html (hunk 3 above)
Comment (actlikewill) [reply peachjam/templates/peachjam/_survey_banner_lg.html]: added

File: peachjam/templates/peachjam/_survey_banner_sm.html (hunk 4 above)
Comment (actlikewill) [reply peachjam/templates/peachjam/_survey_banner_sm.html]: added

**Review by longhotsummer — APPROVED** [review]

**Pull request merged**